pkg/volume/hostpath/host_path_test.go

```diff
@@ -154,8 +155,9 @@ func TestDeleterTempDir(t *testing.T) {
 }
 
 func TestProvisioner(t *testing.T) {
-	tempPath := fmt.Sprintf("/tmp/hostpath/%s", uuid.NewUUID())
-	defer os.RemoveAll(tempPath)
+	dir := "/tmp/hostpath"
```

Comment:
We should probably use [MkTmpdir](https://github.com/kubernetes/kubernetes/blob/cff7e2133982fa5b982a0e15ec768acf1970c105/pkg/volume/gcepd/gce_pd_test.go#L39) to get a unique directory to use for the test.

Reply:
done

But for hostpath_pv, we could not do that, which it is currently decided by the provisioner and may affect users.